src/datepicker/docs/readme.md

```diff
@@ -1,145 +1,150 @@
+  Format of month in year.
+
+* `format-year`
+  _(Default: `yyyy`)_ -
+  Format of year in year range.
+
+* `format-day-header`
+  _(Default: `EEE`)_ -
+  Format of day in week header.
+
+* `format-day-title`
+  _(Default: `MMMM yyyy`)_ -
+  Format of title when selecting day.
+
+* `format-month-title`
+  _(Default: `yyyy`)_ -
+  Format of title when selecting month.  
+
+* `init-date`
+  _(Default: `null`)_ -
+  The initial date view when no model value is specified.
+  
+* `max-date`
+  <i class="glyphicon glyphicon-eye-open"></i>
+  _(Default: `null`)_ -
+  Defines the maximum available date.
+  
+* `max-mode`
+  _(Default: `year`)_ -
+  Set an upper limit for mode.
+  
+* `min-date`
+  <i class="glyphicon glyphicon-eye-open"></i>
+  _(Default: `null`)_ -
+  Defines the minimum available date.
+
+* `min-mode`
+  _(Default: `day`)_ -
+  Set a lower limit for mode.
+
+* `ng-model`
+  <i class="glyphicon glyphicon-eye-open"></i> -
+  The date object
+  
+* `shortcut-propagation`
+  _(Default: `false`)_ -
   An option to disable or enable shortcut's event propagation.
 
- * `template-url`
-  _(Default: 'uib/template/datepicker/datepicker.html')_ :
-  Allows overriding of default template of the datepicker
-
-
-### Popup Settings ###
-
-Options for datepicker can be passed as JSON using the `datepicker-options` attribute.
-Specific settings for the `uib-datepicker-popup`, that can globally configured through the `uibDatepickerPopupConfig`, are:
-
- * `uib-datepicker-popup`
- 	_(Default: 'yyyy-MM-dd')_ :
- 	The format for displayed dates. This string can take string literals by surrounding the value with single quotes, i.e. `yyyy-MM-dd h 'o\'clock'`
-
- * `show-button-bar`
- 	_(Default: true)_ :
- 	Whether to display a button bar underneath the datepicker.
-
- * `current-text`
- 	_(Default: 'Today')_ :
- 	The text to display for the current day button.
-
- * `clear-text`
- 	_(Default: 'Clear')_ :
- 	The text to display for the clear button.
-
- * `close-text`
- 	_(Default: 'Done')_ :
- 	The text to display for the close button.
-
- * `close-on-date-selection`
- 	_(Default: true)_ :
- 	Whether to close calendar when a date is chosen.
-
- * `datepicker-popup-template-url`
-  _(Default: 'uib/template/datepicker/popup.html')_ :
-  Allows overriding of default template of the popup
-
- * `datepicker-template-url`
-  _(Default: 'uib/template/datepicker/datepicker.html')_ :
-  Allows overriding of default template of the datepicker used in popup
-
- * `datepicker-append-to-body`
-  _(Default: false)_:
-  Append the datepicker popup element to `body`, rather than inserting after `datepicker-popup`. For global configuration, use `uibDatepickerPopupConfig.appendToBody`.
-
- * `is-open` <i class="glyphicon glyphicon-eye-open"></i>
-  _(Default: false)_:
+* `show-weeks`
+  _(Default: `true`)_ -
+  Whether to display week numbers.
+
+* `starting-day`
+  _(Default: `0`)_ -
+  Starting day of the week from 0-6 (0=Sunday, ..., 6=Saturday).
```

Comment:
this is more of a feature request, but i've never been a fan of 0-based stuff for calendar - Java had this issue for a LOOOOONG time and finally provided replacements for the Calendar classes....and why JodaTime came into existence.